ui/src/apiContext.ts

```diff
@@ -227,6 +227,8 @@ class FakeHintsApi {
   }
 }
 
+class FakeStudiesAPI {}
```

Comment:
Not sure how important it is to define a full fake api, just added an empty class for now

Reply:
This is the current method we're using to provide fake data for integration testing using Cypress. We'll need to replace these with a more comprehensive solution to support artifact storage but for now, at least, this is where you'd need to add data if you wanted to add tests.

Reply:
Ok, good to know. I have not used Cypress but I'll take a look and try to follow up with some tests.